### PR TITLE
Multimenu

### PR DESCRIPTION
**Pull request by bwrsandman**
Adding menu module for multiple Result Based Management level chains to separated in different menus at the top of Odoo.

```
Progam              | Other Program Top Menu
====================|=======================
                    |
Results             | Results
-------             | -------
Mission             | Mission type 2
Final Result        | Final Result 2
.                   | .
.                   | .
.                   | .
Activity            | Activity 2
                    |
Configuration       | Configuration
-------------       | -------------
Result Levels       | Result Levels (filtered Mission type 2, etc)
.                   | .
.                   | .
.                   | .
```

### CODE REVIEW

File: program_multi_menu/tests/test_program_result_level.py [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+        self.assertFalse(
+            self.registry("program.result.intervention").search(
+                self.cr, self.uid, [('id', '=', self.int)],
+                context=self.context
+            )
+        )
+
+    def test_unlink3(self):
+        """Make sure menus and actions are deleted when the last member of the
+        chain is unlinked"""
```
Comment (ghost) [inline program_multi_menu/tests/test_program_result_level.py:325]: Can you explain in docstring the difference between test_unlink1, test_unlink2 and test_unlink3 please?

**Comment by ghost** [comment]
I tried your module. It seems that in view view_program_result_form, I can't create a record because of the field result_level_id. It is readonly and required.

**Comment by bwrsandman** [comment]
`view_program_result_form` is a template and only visible with elevated permissions. The records should be created by regular users in the respective level menus.

I can definetely work on making creation possible from that menu, but it is not the intended workflow.

File: program_multi_menu/tests/test_program_result_level.py [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+        })
+        self.result_level_1.refresh()
+        self.assertEqual(
+            self.result_level_1.top_level_menu_id.id,
+            menu_id,
+        )
+
+    def test_unlink1(self):
+        """Make sure menus and actions are deleted when the last member of the
+        chain is unlinked"""
```
Comment (bwrsandman) [inline program_multi_menu/tests/test_program_result_level.py:207]: Test the order of deletion from the head (root) down to the tail

File: program_multi_menu/tests/test_program_result_level.py [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+        self.assertFalse(
+            self.registry("program.result.intervention").search(
+                self.cr, self.uid, [('id', '=', self.int)],
+                context=self.context
+            )
+        )
+
+    def test_unlink2(self):
+        """Make sure menus and actions are deleted when the last member of the
+        chain is unlinked"""
```
Comment (bwrsandman) [inline program_multi_menu/tests/test_program_result_level.py:266]: Test the order of deletion from the tail up to the head (root)

File: program_multi_menu/tests/test_program_result_level.py (hunk 1 above)
Comment (bwrsandman) [inline program_multi_menu/tests/test_program_result_level.py:325]: Test mass deletion of an entire chain

Comment (bwrsandman) [inline program_multi_menu/tests/test_program_result_level.py:325]: @dufresnedavid I will update the docstrings, thanks

**Comment by bwrsandman** [comment]
Reset branch back to 2ad274c, @ehdem's commits will be in another PR

**Pull request merged**